plugins/modules/remote_management/redfish/wdc_redfish_info.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2022 Western Digital Corporation
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: wdc_redfish_info
+short_description: Manages WDC UltraStar Data102 Out-Of-Band controllers using Redfish APIs
+description:
+  - Builds Redfish URIs locally and sends them to remote OOB controllers to
+    get information back.
+options:
+  category:
+    required: true
+    description:
+      - Category to execute on OOB controller
+    type: str
+  command:
+    required: true
+    description:
+      - List of commands to execute on OOB controller
+    type: list
+    elements: str
+  baseuri:
+    description:
+      - Base URI of OOB controller.  Must include this or ioms
+    type: str
+  ioms:
+    description:
+      - List of IOM FQDNs for the enclosure.  Must include this or baseuri
+    type: list
+    elements: str
+  username:
+    description:
+      - User for authentication with OOB controller
+    type: str
+  password:
+    description:
+      - Password for authentication with OOB controller
+    type: str
+  auth_token:
+    description:
+      - Security token for authentication with OOB controller
+    type: str
+  timeout:
+    description:
+      - Timeout in seconds for URL requests to OOB controller
+    default: 10
+    type: int
+
+requirements:
+  - dnspython (2.1.0 for Python 3, 1.16.0 for Python 2)
+
+notes:
+  - In the inventory, you can specify baseuri or ioms.
+  - ioms is a list of FQDNs for the enclosure's IOMs.
+  - See sample below.  This can be in your hosts file such as /etc/ansible/hosts.
+  - my_enclosure ioms='["oobm-11-22-33-44-55-66.wdc.com", "oobm-22-33-44-55-66-77.wdc.com"]'
+  - my_enclosure2 baseuri="oobm-00-11-22-33-44-55.wdc.com"
+
+author: Mike Moerk (@mikemoerk)
+'''
+
+EXAMPLES = '''
+- name: Get Simple Update Status
+  community.general.wdc_redfish_info:
+    category: Update
+    command: SimpleUpdateStatus
+    ioms: "{{ ioms }}"
+    username: "{{ username }}"
+    password: "{{ password }}"
```

Comment:
```suggestion
    password: "{{ password }}"
  register: result

- ... a task that uses result ...
```

Reply:
This one I'm not sure about -- currently I don't have an example of a task that uses the result.

Reply:
In that case, you don't need an `_info` module if you never used its result. If this module isn't needed, you should remove it from this PR.

Reply:
I guess I am confused.  The SimpleUpdateStatus command will be useful for end users and management tools to report on the status.  It's just that I'm currently not using it to feed into another Ansible task.  Since it was providing info and not changing anything, I put it into `wdc_redfish_info` rather than `wdc_redfish_command`.  Is this not correct?

Reply:
If you run Ansible with such a task, the output a user gets is
```
ok: [name.of.server]
```
(If you are using a special callback plugin or extra verbosity you can see more, but that's it you get by default.)

So for `_info` module calls you usually register the result and either output it in a `ansible.builtlin.debug` task, or use it in another task.

Reply:
@felixfontein , thanks for the explanation.  I'll have updates on Monday.

Reply:
```suggestion
  register: result
```

Reply:
Ugh, sorry I missed that and thanks for catching it.  Fixed in [1d685aac4bfda44ee40f46af71ec03c2d1889761](https://github.com/ansible-collections/community.general/pull/4885/commits/1d685aac4bfda44ee40f46af71ec03c2d1889761).

I did't commit the suggestion as-is because I don't think we want to delete the `password` line.